src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -404,82 +455,82 @@ protected override void PostStop()
                 // We still need to clean up any remaining transports because handles might be in mailboxes, and for example
                 // Netty is not part of the actor hierarchy, so its handles will not be cleaned up if no actor is taking
                 // responsibility of them (because they are sitting in a mailbox).
-                log.Error("Remoting system has been terminated abrubtly. Attempting to shut down transports");
+                _log.Error("Remoting system has been terminated abrubtly. Attempting to shut down transports");
                 foreach (var t in _transportMapping.Values)
                     t.Shutdown();
             }
         }
 
-        protected override void OnReceive(object message)
+        private void Receiving()
```

Comment:
Default behavior for the `EndpointManager` - didn't change the message-handling much. Only adjustments made were for the differences in signatures on the `EndpointRegistry`.